cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -71,6 +71,11 @@ public MainServer(ServerConfiguration serverConfiguration) {
 	public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
+
+        if(mods.getAvailableMods().isEmpty()) {
+        	logger.warn("No mods were loaded from the different folders. Please check your installation.");
```

Comment:
It already logs which directory are loaded in `ModCollection#loadExternal` `logger.info(directory.toAbsolutePath() + " not found. External mod directory not loaded");` so it would be redundant here.